### PR TITLE
multi: small code fix

### PR DESCRIPTION
**Pull request by yyforyongyu**
Minor code fix

### CODE REVIEW

File: htlcswitch/link.go [hunk 1]
```diff
@@ -516,7 +516,14 @@ func (l *channelLink) Stop() {
 		l.cfg.ChainEvents.Cancel()
 	}
 
-	l.updateFeeTimer.Stop()
+	// Ensure the channel for the timer is drained.
+	if !l.updateFeeTimer.Stop() {
```
Comment (bhandras) [inline htlcswitch/link.go:520]: Nice! Did this actually happen?

**Review by bhandras — APPROVED** [review]
LGTM

File: htlcswitch/link.go (hunk 1 above)
Comment (yyforyongyu) [reply htlcswitch/link.go:520]: Actually I wasn't sure...was playing with `pprof` and saw this code so I made the change😂

**Review by carlaKC — APPROVED** [review]
LGTM

**Review by cfromknecht — APPROVED** [review]
LGTM!

**Pull request merged**